.gitignore

```diff
@@ -24,3 +24,4 @@ valgrind-python.supp
 lcov.info
 netlify_build/
 .nox/
+.vscode/
```

Comment:
In previous discussion we've asked contributors to keep editor specifics out of this file and in their global configuration instead:

https://github.com/PyO3/pyo3/pull/3042#discussion_r1133259224